lib/Form/Block/Divider.php

```diff
@@ -3,8 +3,41 @@
 namespace MailPoet\Form\Block;
 
 class Divider {
+  const DEFAULT_ATTRIBUTES = [
+    'height' => 1,
+    'type' => 'divider',
+    'style' => 'solid',
+    'dividerHeight' => 1,
+    'dividerWidth' => 100,
+    'color' => 'black',
+  ];
+
   public function render($block): string {
-    $classes = isset($block['params']['class_name']) ? " " . $block['params']['class_name'] : '';
-    return '<hr class="mailpoet_divider' . $classes . '" />';
+    $classes = isset($block['params']['class_name']) ? $block['params']['class_name'] : '';
+    $height = $block['params']['height'] ?? self::DEFAULT_ATTRIBUTES['height'];
+    return "<div class='mailpoet_spacer $classes' style='height: {$height}px;'>"
+    . $this->renderDivider($block)
+    . '</div>';
+  }
+
+  private function renderDivider(array $block): string {
+    if (isset($block['params']['type']) && $block['params']['type'] === 'spacer') {
+      return '';
+    }
+    $width = $block['params']['divider_width'] ?? self::DEFAULT_ATTRIBUTES['dividerWidth'];
+    $style = $block['params']['style'] ?? self::DEFAULT_ATTRIBUTES['style'];
+    $dividerHeight = $block['params']['divider_height'] ?? self::DEFAULT_ATTRIBUTES['dividerHeight'];
+    $color = $block['params']['color'] ?? self::DEFAULT_ATTRIBUTES['color'];
+    $height = $block['params']['height'] ?? self::DEFAULT_ATTRIBUTES['height'];
+
+    $dividerStyles = [
+      "border-top-style: $style",
+      "border-top-width: {$dividerHeight}px",
+      "border-top-color: $color",
+      "height: {$height}px",
+      "width: $width%",
```

Comment:
This renders the divider with some height differently than in form editor.
In the form editor, the divider is vertically centred https://d.pr/i/7cj6hO 
On the front end, the divider is displayed on top https://d.pr/i/FkVR7c

I see that this behaviour was not specified in spec. I would personally expect the behaviour we have in form editor - vertically centred.